# result status
int8 FAILED      = -1
int8 UNSET       = 0
int8 SUCCEEDED   = 1

bool available
int8 result
string error_message
float64 data
float64 epsilon
